src/main/java/org/example/algorithms/Recursion example

```diff
@@ -0,0 +1,31 @@
+public class palindrome
```

Comment:
It seems like it is a text file. It should be .java file.

Reply:
Ok